.gitignore

```diff
@@ -147,3 +147,4 @@ test-results
 
 *.ir
 *.meta
+/nbproject/private/
```

Comment:
This gitignore should only contain entries related to the project. For files related to your machine else please use global gitignore https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Reply:
Removed in [d63c0fb](https://github.com/enso-org/enso/pull/3509/commits/d63c0fb1e9d6b34386a4e8d45b6dbd2d13b01c1e) - I am sorry for this accidental change. I didn't do it - my IDE did it behind the scene!